.github/workflows/tests.yml

```diff
@@ -169,3 +169,202 @@ jobs:
           export -f _run_tests
 
           find src/Symfony -mindepth 2 -type f -name phpunit.xml.dist -not -wholename '*/Bridge/PhpUnit/*' -print0 | xargs -0 -n1 dirname | sort | parallel _run_tests
+
+  high_deps:
+    name: PHPUnit (7.4, high deps)
+    runs-on: Ubuntu-20.04
+
+    env:
+      SYMFONY_DEPRECATIONS_HELPER: weak
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          coverage: "none"
+          ini-values: "memory_limit=-1"
+          php-version: "7.4"
+          extensions: amqp, apcu, igbinary, intl, mbstring, memcached, mongodb, redis
+          tools: flex
+
+      - name: Configure composer
+        run: |
+          SYMFONY_VERSION=$(grep -m1 SYMFONY_VERSION .travis.yml | grep -o '[0-9.x]*')
+          SYMFONY_REQUIRE=">=$SYMFONY_VERSION"
+          COMPOSER_HOME="$(composer config home)"
+          composer self-update
```

Comment:
This is not needed as far as I know.

Reply:
indeed. `shivammathur/setup-php` already takes care of installing an uptodate one

Reply:
FYI: This line is C&P from existing GHA jobs. I agree that we should remove it everywhere.